### PR TITLE
Add TurboSnap support for Vite

### PR DESCRIPTION
**Pull request by IanVS**
I'm on a mission to use turbosnap in my own project, which is built using storybook-builder-vite.  It's going to take a few different pieces:

- A rollup plugin to generate something similar to `preview-stats.json` from webpack.  
- Some changes to storybook-builder-vite (https://github.com/eirslett/storybook-builder-vite/pull/239)
- A few tweaks here to chromatic, to account for a few differences between webpack and vite.

### CODE REVIEW

File: bin-src/lib/getDependentStoryFiles.ts [hunk 1]
```diff
@@ -145,9 +150,8 @@ export async function getDependentStoryFiles(
   function files(moduleName: string) {
     const mod = modulesByName[moduleName];
     if (!mod) return [moduleName];
-    return mod.modules && MULTI_MODULES.test(mod.name)
```
Comment (IanVS) [inline bin-src/lib/getDependentStoryFiles.ts:148]: I'd like to understand why this regex test was needed.  The naming for the vite modules does not include it.

**Comment by IanVS** [comment]
I've been running turbosnap on my own repo with these (and other) changes `patch-package`ed in to my project, and it seems to be working great!  🎉 .  I've marked this as ready for review, and I think it could be merged any time without impacting existing webpack users of turbosnaps.

**Review by ghengeveld — APPROVED** [review]

**Comment by yannbf** [comment]
Hey @IanVS thank you so much for all of your efforts! As always the work you've been doing is incredible and we're incredibly thankful for each and every contribution you make! ♥️

Thanks for helping Storybook and its  ecosystem thrive!

**Comment by ghengeveld** [comment]
Released in [6.5.0](https://github.com/chromaui/chromatic-cli/blob/main/CHANGELOG.md#650---2022-02-21).

**Pull request merged**